src/ppx_import.ml

```diff
@@ -207,41 +197,49 @@ let rec core_type_of_type_expr ~subst type_expr =
       {typ with ptyp_desc = Ptyp_constr (lid, args)}
     | _ -> assert false
     | exception Not_found ->
-      Typ.constr {txt = longident_of_path path; loc = !default_loc} args )
+      Ppxlib.Ast_helper.Typ.constr
+        {txt = longident_of_path path; loc = !Ppxlib.Ast_helper.default_loc}
+        args )
   | Tvariant {row_fields; _} ->
     let fields =
       row_fields
       |> List.map (fun (label, row_field) ->
-             let label = Location.mknoloc label in
+             let label = Ocaml_common.Location.mknoloc label in
```

Comment:
What do you think about inlining a `mknoloc` function instead of using the compiler-libs one?

In that context: it might even be worth filing an issue on ppxlib to add a `mknoloc` function to `Ppxlib.Location`. I think on one hand not having one is intentional so that people are aware that using a ghost location isn't standard, but on the other hand I've already seen several projects where that's necessary.